reedsolomon/node.go

```diff
@@ -54,7 +53,6 @@ func (n *Node) AddRedundantNode(nd format.Node) {
 
 	n.cache = nil
 	n.recovery = append(n.recovery, &format.Link{
-		Name: strconv.Itoa(len(n.recovery)), // TODO Do we need a name?
```

Comment:
Something generic like `Recovery Link`?

Reply:
Pls expand on that?

Reply:
We can do special `type RecoveryLink`, but for now I don't see any reasoning for this, nor use cases, except for saving some bytes in serialized form.